src/main/java/io/github/syst3ms/skriptparser/expressions/ExprAnswer.java

```diff
@@ -0,0 +1,50 @@
+package io.github.syst3ms.skriptparser.expressions;
+
+import io.github.syst3ms.skriptparser.Parser;
+import io.github.syst3ms.skriptparser.lang.Expression;
+import io.github.syst3ms.skriptparser.lang.TriggerContext;
+import io.github.syst3ms.skriptparser.lang.lambda.SectionValue;
+import io.github.syst3ms.skriptparser.parsing.ParseContext;
+import io.github.syst3ms.skriptparser.sections.SecAsk;
+
+/**
+ * Get back the last entered answer in a {@link SecAsk} section.
+ * This will return the last line of the console.
+ *
+ * @name Last Answer
+ * @type EXPRESSION
+ * @pattern [the] [last] (answer|response)
+ * @since ALPHA
+ * @author Mwexim
+ */
+public class ExprAnswer extends SectionValue<SecAsk, String> {
+
+    static {
+        Parser.getMainRegistration().addExpression(
+                ExprAnswer.class,
+                String.class,
+                true,
+                "[the] [last] (answer|response)"
+        );
+    }
+
+    @Override
+    public String toString(TriggerContext ctx, boolean debug) {
```

Comment:
This method should be at the end of the class, because all ``toString`` methods are not important and so they're at the end.